File: RRMemberRegister/ClientApp/src/components/member/Member.js

/* Klass med data om en Member */
class Member {
    constructor() {
        this.id = 0;
        this.fornamn = "";
        this.efternamn = "";
        this.adress = "";
        this.postnummer = "";
        this.postort = "";
        this.skapatdatumasstring = "";
    }

    /* Get/Set metoder */
    get Id() {
        return this.id;
    }

    set Id(newId) {
        this.id = newId;
    }

    get Fornamn() {
        return this.fornamn;
    }

    set Fornamn(newFornamn) {
        this.fornamn = newFornamn;
    }

    get Efternamn() {
        return this.efternamn;
    }

    set Efternamn(newEfternamn) {
        this.efternamn = newEfternamn;
    }

    get Adress() {
        return this.adress;
    }

    set Adress(newAdress) {
        this.adress = newAdress;
    }

    get Postnummer() {
        return this.postnummer;
    }

    set Postnummer(newPostnummer) {
        this.postnummer = newPostnummer;
    }

    get Postort() {
        return this.postort;
    }

    set Postort(newPostort) {
        this.postort = newPostort;
    }

    get Skapatdatum() {
        return this.skapatdatumasstring;
    }

    set Skapatdatum(newDatum) {
        this.skapatdatumasstring = newDatum;
    }

    get Namn() {
        return "" + this.fornamn + " " + this.efternamn;
    }

    /* Return information om Member objektet */
    ToString() {
        return "Id: " + this.id +
            ", Namn: " + this.fornamn + " " + this.efternamn +
            ", Adress: " + this.adress +
            " Postadress: " + this.postnummer + " " + this.postort +
            " Skapat datum: " + this.skapatdatumasstring;
    }
}

export { Member };

File: RRMemberRegister/Models/Member.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace MemberRegister.Models
{
    public class Member
    {
        [Key]
        public int Id { get; set; }

        [DisplayName("Förnamn")]
        public string Fornamn { get; set; }

        [DisplayName("Efternamn")]
        public string Efternamn { get; set; }

        [DisplayName("Adress")]
        public string Adress { get; set; }

        [DisplayName("Postnummer")]
        public string Postnummer { get; set; }

        [DisplayName("Postort")]
        public string Postort { get; set; }

        [DisplayName("Skapat datum")]
        public DateTime Skapatdatum { get; set; }

        [DisplayName("Datum för senaste uppdateringen")]
        public DateTime Senastuppdateraddatum { get; set; }

        [NotMapped]
        [ScaffoldColumn(false)]
        [DisplayName("Skapat datum")]
        public string Skapatdatumasstring { get; set; }

        [NotMapped]
        [ScaffoldColumn(false)]
        [DisplayName("Namn")]
        public string Namn {
            get 
            {
                return this.Fornamn + " " + this.Efternamn;
            }
        }
        
        public override string ToString()
        {
            StringBuilder strBuild = new StringBuilder("Id: " + this.Id);
            strBuild.Append(System.Environment.NewLine);

            strBuild.Append(this.Namn);
            strBuild.Append(System.Environment.NewLine);

            strBuild.Append(this.Adress);
            strBuild.Append(System.Environment.NewLine);

            strBuild.Append(this.Postnummer);
            strBuild.Append(" ");
            strBuild.Append(this.Postort);            

            strBuild.Append(System.Environment.NewLine);
            strBuild.Append("Skapat datum: " + this.Skapatdatumasstring);

            strBuild.Append(System.Environment.NewLine);
            strBuild.Append("Senast uppdaterad: " + this.Senastuppdateraddatum.ToShortDateString());

            return strBuild.ToString();
        }
    }
}